### ISSUE TITLE
Unable to reproduce the results in the paper

### ISSUE THREAD

**Issue by VickiCui** [zinengtang/VidLanKD#2]
Hi, 
I used the student model you released and fine-tuned it on GLUE. My experimental sitting is consistent with that in the paper “train 3 epochs with a learning rate of 1e-4 and a batch-size of 32”. Unfortunately, my experimental results are much worse than those in the paper.

My results (The BERT_baseline is the one from Vokenization):
![image](https://user-images.githubusercontent.com/20258329/142381643-3fac587b-95bc-40e4-b58b-cbce1dc79d85.png)
Results in paper:
![image](https://user-images.githubusercontent.com/20258329/142381305-e1eb0658-a20c-4fbc-8a31-6245f861d397.png)

In addition, I also noticed that the random seed has a great impact on the performance of VidLanKD on QNLI and QQP tasks.
I wonder why? Do you have any other settings that need to pay attention to during the fine-tuning?

**Comment by zinengtang** [comment]
The student model I just released is only based on NST. My reproduced
result of this model is 92.2, 89.8, 89.5, 83.4 respectively. Let me check
the issue of this checkpoint and release the full model nst+crd tomorrow.

**Comment by VickiCui** [comment]
> The student model I just released is only based on NST. My reproduced result of this model is 92.2, 89.8, 89.5, 83.4 respectively. Let me check the issue of this checkpoint and release the full model nst+crd tomorrow.

OK, thanks for your prompt reply.

**Comment by VickiCui** [comment]
I repeated the experiment under three random seeds and recorded the results of VidLanKD:
![image](https://user-images.githubusercontent.com/20258329/142559147-0cacb9ec-78cd-4920-9944-90aefed6d5d2.png)
I think there must be something wrong.

**Comment by zinengtang** [comment]
@VickiCui I think one reason might be that the model is trained on fp16 which might not be always stable. 
https://drive.google.com/drive/folders/1zpwjej5SuuMJ65YU02x_T7kxbd7Ed-j_?usp=sharing I selected other checkpoint of nst base model. Let me test other models to see if they are stable and then release.
Feel free to open other issues if you have still have issues.
